### ISSUE TITLE
Cli - Include cluster name (along with project name)  when listing projects for default project selection for log in command.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11466]
rancher version - v1.0.0-alpha5

Include cluster name (along with project name)  when listing projects for default project selection for log in command.

Currently when logging in for rancher cli , project listing does not include clustername

```
Sangeethas-MBP:rancher-v1.0.0-alpha5 sangeethahariharan1$ ./rancher login -t token-**** --cacert ~/ca-cert --name test1 https://***/v3
Select your default Project:
0 cluster-f2kr2:project-4g6pr Default
1 cluster-f2kr2:project-snc8m mynewp1
2 cluster-j66nx:project-bpsg6 Default
0
```

**Comment by dramich** [comment]
Available with v1.0.0-alpha7

**Comment by sangeethah** [comment]
Tested with rancher version - v1.0.0-alpha7
Cluster name is presented  (along with project name) when listing projects for default project selection for log in command
